src/lumigo_tracer/lambda_tracer/spans_container.py

```diff
@@ -246,7 +245,7 @@ def add_exception_to_span(  # type: ignore[no-untyped-def]
             message = str(message)
         span["error"] = SpansContainer._create_exception_event(
             exc_type=exception.__class__.__name__,
-            message=message,
+            message=lumigo_dumps(message),
```

Comment:
message is string. Do you double escape here?

Reply:
I want to limit its size